<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Agregar</class>
 <widget class="QDialog" name="Agregar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>232</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="titulo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>221</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Nuevo articulo</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="btns_option">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>231</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_nombre">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Producto:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="in_nombre">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="cursor">
       <cursorShape>IBeamCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="toolTipDuration">
       <number>-1</number>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>32767</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string>Ingrese nombre de producto</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_cant">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Cantidad:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="in_cant">
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>32767</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string>Ingrese valor numerico</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_precio">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Precio:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="in_precio">
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>32767</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string>Ingrese valor numerico</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_descrip">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Descripcion:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="in_descrip">
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>32767</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string>Ingrese una breve descripcion</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
